addons/binding/org.openhab.binding.velbus/ESH-INF/thing/thing-types.xml

```diff
@@ -8,20 +9,174 @@
 		<description>This bridge represents a Velbus Serial-interface</description>
 		<config-description>
 			<parameter name="port" type="text">
+				<context>serial-port</context>
 				<label>Serial port</label>
 				<description>Select serial port (COM1, /dev/ttyS0, ...)</description>
 				<required>true</required>
 			</parameter>
 		</config-description>
 	</bridge-type>
 
-	<thing-type id="vmb4ryld">
+	<thing-type id="vmb1bls">
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="bridge" />
 		</supported-bridge-type-refs>
 
-		<label>VMB4RYLD</label>
-		<description>4 channel relay module with direct load connections for VELBUS system</description>
+		<label>VMB1BLS</label>
+		<description>1-channel blind control module for universal mounting</description>
+		<channels>
+			<channel id="CH1" typeId="rollershutter" />
+		</channels>
+
+		<config-description>
```

Comment:
A lot of your config-descriptions are duplicates please create reusable ones as a separate file in the `config` directory, see https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#configuration-descriptions

You can reference them like it is done in the [tradfri binding](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml#L29)

Reply:
Modified in commit db6361f